WordPress/dashicons
# Split build process for Gutenberg icons

This PR fixes #212. It changes the build process so icons in the folder `sources/svg/gutenberg` are separate, and not included in the icon font. They are included in the React component, though, which is what Gutenberg uses. That way icons can still be added to the font for the normal 4.9 process, without necessarily including all the new icons that Gutenberg will be adding.

Steps to test:

- Verify that the icon font doesn't include the new icons, and that the icons present are at their correct unicode locations.
- Verify that icons _are_ present in the react component.
